extension/src/service/web-component-service.js

```diff
@@ -24,6 +24,10 @@ function manualCapture (manualCaptureRequestObj) {
   return callAdyen(`${config.adyen.legacyApiBaseUrl}/capture`, manualCaptureRequestObj)
 }
 
+function cancelPayment (cancelRequestObj) {
+  return callAdyen(`${config.adyen.legacyApiBaseUrl}/cancel`, cancelRequestObj)
```

Comment:
if we have no internet access and failed this call?

Reply:
Then it will not be cancelled and the whole update will fail. The shop has to repeat.